test/unit/translations/maps/linkedPhysicalIndexToValueMap.spec.js

```diff
@@ -0,0 +1,217 @@
+import { LinkedPhysicalIndexToValueMap as IndexToValueMap } from 'handsontable/translations';
+
+it('should work with get, and set functions properly', () => {
+  const indexToValueMap = new IndexToValueMap();
+
+  indexToValueMap.setValueAtIndex(0, 2);
+  indexToValueMap.setValueAtIndex(1, 1);
+  indexToValueMap.setValueAtIndex(2, 0);
+
+  expect(indexToValueMap.getValues()).toEqual([]);
+  expect(indexToValueMap.getLength()).toBe(0);
+
+  indexToValueMap.init(3);
+
+  expect(indexToValueMap.indexedValues).toEqual([null, null, null]);
+  expect(indexToValueMap.getValues()).toEqual([]); // Returns only non-default values.
+  expect(indexToValueMap.getLength()).toBe(0); // Returns number of non-default values.
+
+  indexToValueMap.setValueAtIndex(0, 2);
+  indexToValueMap.setValueAtIndex(1, 1);
+  indexToValueMap.setValueAtIndex(2, 0);
+
+  expect(indexToValueMap.indexedValues).toEqual([2, 1, 0]);
+  expect(indexToValueMap.getValues()).toEqual([2, 1, 0]);
+  expect(indexToValueMap.getLength()).toBe(3);
+
+  indexToValueMap.setValues([1, 2, 0]);
+
+  expect(indexToValueMap.indexedValues).toEqual([1, 2, 0]);
+  expect(indexToValueMap.getValues()).toEqual([1, 2, 0]);
+  expect(indexToValueMap.getLength()).toBe(3);
+});
+
+it('should init map properly when passing function as initialization property', () => {
+  const indexToValueMap = new IndexToValueMap(index => ({ key: index }));
+
+  indexToValueMap.init(3);
+
+  expect(indexToValueMap.indexedValues).toEqual([{ key: 0 }, { key: 1 }, { key: 2 }]);
+  expect(indexToValueMap.getValues()).toEqual([]); // Returns only non-default values.
```

Comment:
The comment `// Returns only non-default values.` can be confusing here. As in the line above the `indexedValues` are filled with some non-null values which can be read as non-default ones. But in fact, `getValues` returns an empty array because for that Collection the `orderOfIndexes` array is a key.

Reply:
Removed within: https://github.com/handsontable/handsontable/pull/7276/commits/069622ac53b024960a5510a69df4b19e42cb672f and https://github.com/handsontable/handsontable/pull/7276/commits/8494f6bf9c2adba659f81b6bf8a60cc43a95b8d8